<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DefineSampleBoxDialog</class>
 <widget class="QDialog" name="DefineSampleBoxDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Minimum Corner (X, Y, Z)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxMinX">
        <property name="prefix">
         <string>X=</string>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-9999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxMinY">
        <property name="prefix">
         <string>Y=</string>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-9999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxMinZ">
        <property name="prefix">
         <string>Z=</string>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-9999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Maximum Corner (X, Y, Z)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxMaxX">
        <property name="prefix">
         <string>X=</string>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-9999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxMaxY">
        <property name="prefix">
         <string>Y=</string>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-9999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxMaxZ">
        <property name="prefix">
         <string>Z=</string>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-9999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Spinner Controls</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QSpinBox" name="spinBoxDecimals">
        <property name="suffix">
         <string/>
        </property>
        <property name="prefix">
         <string>DP=</string>
        </property>
        <property name="maximum">
         <number>9</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxIncrement">
        <property name="prefix">
         <string>Inc=</string>
        </property>
        <property name="decimals">
         <number>9</number>
        </property>
        <property name="minimum">
         <double>0.000001000000000</double>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxRange">
        <property name="prefix">
         <string>Range=</string>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>99999999.000000000000000</double>
        </property>
        <property name="value">
         <double>9999.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
